web/src/drafts.ts

```diff
@@ -474,6 +474,26 @@ export function filter_drafts_by_compose_box_and_recipient(
     return _.pick(drafts, narrow_drafts_ids);
 }
 
+export function get_last_draft_based_on_compose_state():
+    | (LocalStorageDraft & {id: string})
+    | undefined {
+    if (compose_state.message_content()) {
+        return undefined;
+    }
```

Comment:
Why is this check here? It feels to me like this should be the caller's responsibility.

Reply:
I can't remember at this point, and it makes sense to me to make it the caller's responsibility so I'll just move it there